src/helm/benchmark/adaptation/adapters/in_context_learning_adapter.py

```diff
@@ -224,16 +224,28 @@ def construct_example_prompt(self, instance: Instance, include_output: bool, ref
         # Input
         result: str = self.adapter_spec.input_prefix + (instance.input.text or "") + self.adapter_spec.input_suffix
 
-        # References (optionally) and output
-        output: str
+        if include_output:
```

Comment:
It'd be helpful if you could go through (for next time) and add some general guiding comments like "this piece of code is moved up from below"

Reply:
Ah, the diff was a bit confusing. Yes, I will make sure to do that next time.